readme.md

```diff
@@ -0,0 +1,67 @@
+Misc-deps is a collection of various static dependencies used by [NVDA](https://github.com/nvaccess/nvda).
+
+# Updating contents
+
+## include
+
+### Adobe Acrobat Access
+
+1. Download the latest [Adobe Acrobat SDK](https://developer.adobe.com/console/servicesandapis)
+1. Extract the zip
+1. Copy the file from `Adobe\Acrobat DC SDK\Version 1\AccessibilitySupport\AcrobatAccess.idl`
+
+### iSimpleDOM
+
+MSAA interfaces for gecko / firefox.
+
+Update from [here](https://github.com/mozilla/gecko-dev/tree/master/accessible/interfaces/msaa)
+
+### mathPlayer
+
+Project has been abandoned, updates are unlikely.
+In NVDA this will be abandoned soon in favour of MathCAT.
+
+## python
+
+### ftdi2
+
+Needed for papenmeier displays.
+Should be replaced with [pyftdi](https://github.com/eblot/pyftdi) as a pip dependency.
+This will require a significant rewrite of the papenmeier display driver.
+
+### Updating brlapi
+
+Used for BrlTTY.
+
+1. Download the latest `brltty-win-X.X-libusb.zip` from the [BrlTTY website](https://brltty.app/download.html).
+This is the 32bit version.
+1. Extract the zip.
+1. Update the following files
+    - `brlapi.cp38-win32.pyd` from `brltty-win-X.X-0-libusb\Python\Lib\site-packages\brlapi.cp38-win32.pyd`

```

Comment:
Isn't this a Python 3.8 specific version? I'm afraid we have to provide a custom build instead. We had to do that in the past as well.

Reply:
Ideally, we have a brlapi module that is build against the minimal API, so the module is no longer tied to a specific Python version. I opened a discussion on the mailing list: https://brltty.app/pipermail/brltty/2023-September/020094.html

Reply:
I haven't yet managed to build brlapi for Python 3.11. The [instructions for Windows](https://brltty.app/doc/Windows.html) about MSYS1 I used in the past no longer work on my system. May be @DaveMielke might be able to assist with this.

Reply:
Pull request https://github.com/brltty/brltty/pull/429 is now merged, which brings this a step nearer.

Reply:
Yep! I'll be receiving a build from Dave soon which includes this commit

Reply:
There's still some required work on BRLTTY to use the stable ABI, I've emailed Dave about this